src/utils/hints/transactionsHints.js

```diff
@@ -27,4 +27,25 @@ export const transactionsHints = {
   gAAttachTx: 'Converts a regular, plain-old account into a generalized account by attaching a smart contract to it that has to implement a special function to authorize transactions.',
   gAMetaTx: 'This transaction type must be used in order to execute transactions for a generalized account where the authorization is handled by a smart contract. The actual transaction is wrapped and will only be executed if the provided auth data is valid according to the smart contract that verifies it.',
   payingForTx: 'Covers the fees for another user. It wraps any kind of other transaction types.',
+  keyblockHeight: 'Keyblock height when the transaction was mined.',
+  keyblockConfirmations: 'Amount of keyblocks that were added since the transaction was produced.',
+  status: 'Status of the transaction.',
+  microblockHash: 'Microblock in which the transaction is included.',
+  time: 'Exact date and time when the transaction was mined.',
+  fee: 'Fees paid for the transaction.',
+  nonce: 'Value of the nonce for the transaction. Nonce serves as a one-time authenticator per each transaction.',
+  apiLinks: 'Links to sources of the transaction data.',
+  sender: 'Account that dispatched the transaction.',
```

Comment:
Is this for the usual `SpendTx` ? Then I would change:

```suggestion
  sender: 'Account that sent AE coins.',
```

Reply:
yes, fixed